### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Mekuseo**
- Added any missing JSX elements to your project (without adding extra functionality)
- My final design matched Zeplin's design (this includes all elements + the same styling)
- I used the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -2,3 +2,338 @@
...
+  display: flex;
+  gap: 20px;
+  align-items: center;
+  padding: 10px;
+  border-radius: 5px;
+  transition: all 0.3s ease;
+  text-decoration: none;
+}
+
+.nav-item h1 {
```
Comment (binhussen) [inline src/App.css:40]: - Great Job For having this beautiful UI 👍👍 however you have to modify it to the given [zepline](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) design as it is part of the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md) 👌
### here is your design

Screenshot from 2023-03-03 15-53-21

### here is [zepline](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) design
Screenshot 2023-03-03 at 12 02 18 PM

**Review by binhussen — CHANGES_REQUESTED** [review]
.

File: src/App.css [hunk 2]
```diff
@@ -2,3 +2,338 @@
...
+.nave-bar {
+  max-width: 1200px;
+  display: flex;
+  padding: 20px;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  border-bottom: 1px solid #e8e8e8;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
```
Comment (binhussen) [inline src/App.css]: - Great Job For having this beautiful UI 👍👍 however you have to stick it to the given [zepline](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) design as it is part of the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md) 👌
### here is your design
Screenshot from 2023-03-03 15-53-11
Screenshot from 2023-03-03 15-53-21


### here is [zepline](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) design
Screenshot 2023-03-03 at 12 02 18 PM

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @Mekuseo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Used Git flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @Mekuseo          , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  Keep up the good work 🥂 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯 
- [x] linters are passing
- [x] Professional README 💯 
- [x] Add and remove functionality is working as intended 💯 
- [x] Nice and Neat UI design. 🥇 









 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]

**Pull request merged**